assets/src/edit-story/components/canvas/useCanvasGlobalKeys.js

```diff
@@ -19,27 +19,24 @@
  */
 import { v4 as uuidv4 } from 'uuid';
 import { useCallback } from 'react';
-import { renderToStaticMarkup } from 'react-dom/server';
 
 /**
  * Internal dependencies
  */
 import { useStory } from '../../app';
-import useClipboardHandlers from '../../utils/useClipboardHandlers';
+import useGlobalClipboardHandlers from '../../utils/useGlobalClipboardHandlers';
```

Comment:
IMHO, copy/paste is a clearly enough delineated concept that we can keep it separate from keys. There's already `useCanvasKeys`. Maybe the clone would better fit there? LMK your thoughts.

Reply:
Yes, could be. I actually moved it back and forth a few times since sometimes it seemed like all the element handlers should be in one place, other times it seemed to make sense to separate copy/paste.

Will move cloning to the canvas keys, restore the copy-paste separation and let's see how it seems then.